src/main/java/net/minecraftforge/oredict/ItemPredicate.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.oredict;
+
+import java.util.Iterator;
+
+import net.minecraft.item.ItemStack;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterators;
+
+public class ItemPredicate implements Predicate<ItemStack>, Iterable<ItemStack>
+{
+    private final ItemStack item;
+
+    public ItemPredicate(ItemStack item)
+    {
+        this.item = item.copy();
+    }
+
+    @Override
+    public boolean apply(ItemStack input)
+    {
+        return OreDictionary.itemMatches(item, input, false);
+    }
+
+    @Override
+    public Iterator<ItemStack> iterator()
+    {
+        return Iterators.singletonIterator(item.copy());
```

Comment:
While ```OreDictionary.itemMatches``` make no use of the NBT part, perhaps I can discarding it at construction.